### PR TITLE
changes to master branch for RHCOS 4.9 builds

### PR DESCRIPTION
**Pull request by miabbott**
`gitmodules: use testing-devel branch for fedora-coreos-config`

---

`gitmodules: use testing-devel branch for fedora-coreos-config`

Also bump submodule to HEAD of testing-devel

```
$ git shortlog --no-merges 6f39113b0bd0e40d783edcfedc69b93b0492b096..43eb8766112420a2a24d3f872da088bbd752aa48
Andy McCrae (1):
      Move comments in systemd services to own line

Colin Walters (3):
      coreos-boot-mount-generator: Move all "exit 0" checks to early on
      udev/90-coreos-device-mapper: Create label links in real root too
      coreos-boot-mount-generator: Always use mpath for /boot if rd.multipath

Dusty Mabe (1):
      overlay: initramfs teardown: add support for coreos.force_persist_ip karg

Jonathan Lebon (10):
      ignition-ostree-growfs: don't conditionalize on root= karg
      ignition-ostree-growfs: fix dependency to sysroot mount
      overlay: convert more unit descriptions to Title Case
      coreos-gpt-setup: log whether sgdisk was called
      ignition-ostree-uuid-root: also run if root= is provided
      live-generator: don't call `man bootup`
      Add support for multipath on firstboot
      coreos-gpt-setup: add support for multipath
      ignition-ostree-growfs: add support for multipath
      udev/90-coreos-device-mapper: ignore DM_ACTIVATION
```

---

`rhcos-packages: drop pin for 8.4 Beta`

### CODE REVIEW

**Comment by miabbott** [comment]
Locally built successfully; still running `cosa kola run`

**Review by jlebon — APPROVED** [review]
LGTM overall!

> gitmodules: use testing-devel branch for fedora-coreos-config

Shouldn't we also bump the submodule to make that true? Right now, it's still pointing to the tip of the `rhcos-4.8` branch.

**Comment by miabbott** [comment]
> Shouldn't we also bump the submodule to make that true? Right now, it's still pointing to the tip of the `rhcos-4.8` branch.

Yup, good point.  Will update.

**Comment by miabbott** [comment]
Might need this https://github.com/openshift/release/pull/19451

**Comment by miabbott** [comment]
With the submodule bump, this locally built + tested successfully.  And with openshift/release#19451 merged, let's see if we can get the jobs to pass...

/retest

**Comment by jlebon** [comment]
Hmm, I'm not seeing the submodule bump here. Forgot a `git add`? :)

**Comment by miabbott** [comment]
> Hmm, I'm not seeing the submodule bump here. Forgot a `git add`? :)

Huh...let me try again

**Review by jlebon — APPROVED** [review]
/lgtm
